.gitattributes

```diff
@@ -1,6 +1,9 @@
+/docs export-ignore
```

Comment:
Docs should probably be distributed with the package, but no strong feelings, especially on such a tiny library

Reply:
https://github.com/doctrine/cache/blame/master/.gitattributes#L1

:)

Reply:
Fair enough 👍